components/cards/Bounty.vue

```diff
@@ -171,3 +171,10 @@ export default {
   },
 }
 </script>
+
```

Comment:
I don't think this is used anywhere

Reply:
you mean the component? It is used in components/list/Bounty